server/helpers/custom-validators/videos.ts

```diff
@@ -60,6 +61,7 @@ function isVideoNameValid (value: string) {
 
 function isVideoTagValid (tag: string) {
   return exists(tag) && validator.isLength(tag, VIDEOS_CONSTRAINTS_FIELDS.TAG)
+                     && !isStopWord(tag)
```

Comment:
I don't think for tag is good,
If user would create a tag for `the facebook` why we must impose `facebook` ?
And that maybe useless cause I think people are intelligent and will not put a useless word in tags.

Reply:
:laughing: https://peertube2.cpy.re/search?tagsOneOf=the

To answer your question, a `the facebook` tag would be deemed valid by the function. Having two seperate tags `the` and `facebook` would reject `the`.